python/libraries/websocket.py

```diff
@@ -94,6 +116,18 @@ async def send_msg(message):
         print(f"[{url}] {Fore.BLUE}Closed.{Style.RESET_ALL}")
     return none
 
+# https://limecoda.com/how-to-build-basic-websocket-server-python/
+async def createServer(options, port):
+    async def server(websocket, path):
+        print(websocket)
+
+    # Create websocket server
+    start_server = websockets.serve(server, "localhost", port)
+
+    # Start and run websocket server forever
+    asyncio.get_event_loop().run_until_complete(start_server)
+    asyncio.get_event_loop().run_forever()
```

Comment:
```suggestion
    await start_server
```
perhaps?

Reply:
It stops immediately when I do that.

Reply:
```suggestion
    server = await start_server
    await server.wait_closed()
```

Perhaps? Per [documentation](https://websockets.readthedocs.io/en/stable/api/server.html),

> Awaiting `serve()` yields a `WebSocketServer`. This instance provides `close()` and `wait_closed()` methods for terminating the server and cleaning up its resources.

[`wait_closed()`](https://websockets.readthedocs.io/en/stable/api/server.html#websockets.server.WebSocketServer.wait_closed) is awaitable